app/helpers/exports_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ExportsHelper
+  VULNERABLE_CHARACTERS = %w[= + - @].freeze
+
+  def sanitize(value)
+    return value unless value.is_a?(String)
+
+    value.start_with?(*VULNERABLE_CHARACTERS) ? "'#{value}" : value
```

Comment:
This is still rendering the `'` in the CSV
<img width="352" alt="Screenshot 2022-06-17 at 14 38 15" src="https://user-images.githubusercontent.com/5216/174309583-43c565fa-0c50-4d6d-bedd-5c38cd04bb7e.png">